dips/dip-4.md

```diff
@@ -152,7 +152,9 @@ RawTransaction {
 ```
 
 
-## Refunds
+## Refunds (DEPRECATED)
```

Comment:
Why leave all the deprecated stuff in here?  Can we just version the doc like this: https://dip.diem.com/dip-1/#versions